zkevm-circuits/src/mpt_circuit/account_leaf.rs

```diff
@@ -142,9 +142,10 @@ impl<F: Field> AccountLeafConfig<F> {
                 config.is_mod_extension[is_s.idx()] = cb.query_bool();
             }
 
+            // Constraint 1
```

Comment:
My intention is to name the constraints so that reading the MPT specs will be easier (the names will be used there), however, I just started with that, but I plan to equip all the constraints with such names.

Reply:
Actually coming from you @adria0 :)

Reply:
I see the reasoning now. It's a good idea to add anchors to better reference the spec. 
It would be great to add a one-sentence description of the constraint.